app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="devise-container">
+  <h2>Log in</h2>
+
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name), html: { class: "signup-form" } ) do |f| %>
+    <div class="field">
+      
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email" %>
+    </div>
+
+    <div class="field">
+   
```

Comment:
- [ ] Great job working on the wireframe but I think it needs more improvement to be like the given wireframe since the logging button should have the same width as the inputs so kindly update it, ✅💯⭐ and please make the signup page the same as well  ♻

- Your
![image](https://github.com/ShaliniNayan/Blog-app/assets/58553711/30935c26-559e-4093-9f0c-039219bf83d1)


- Given
![image](https://github.com/ShaliniNayan/Blog-app/assets/58553711/be212d76-1bf7-492e-8396-f7438b8ea7b9)




![image](https://github.com/ShaliniNayan/Blog-app/assets/58553711/5e30b012-bb5c-4409-b0e8-50d486ec9156)